NEWS.md

```diff
@@ -1,46 +1,63 @@
 # gt (development version)
 
-## New features
+This release focuses on improvements to two main areas:
+
+1. Better RTF output support:
+  * RTF table generation is now fit for use following an extensive rewrite of internal RTF rendering functions.
```

Comment:
These bullets need to be indented two spaces more (4 total) in order for Markdown to treat them as nested

Reply:
Of course... thank you.